azure/services/resourceskus/client.go

```diff
@@ -57,7 +57,7 @@ func (ac *AzureClient) List(ctx context.Context, filter string) ([]compute.Resou
 	ctx, _, done := tele.StartSpanWithLogger(ctx, "resourceskus.AzureClient.List")
 	defer done()
 
-	iter, err := ac.skus.ListComplete(ctx, filter)
+	iter, err := ac.skus.ListComplete(ctx, filter, "true")
```

Comment:
I couldn't find documentation for this parameter and passing empty string was giving me errors, so I'm not sure what the correct value should be here.

Reply:
seems like it's supposed to be a string-bool as you eventually figured? 

the examples in azure-rest-api-specs seem to use "true" like you did here, did this not work?

Reply:
`"true"` works indeed. but if it's string-bool, I don't know, perhaps it should be `"false"`?

Reply:
So it seems this is about https://docs.microsoft.com/en-us/rest/api/extendedlocation/, so I guess it make sense to put `"true"` here to avoid potentially limiting users, though I don't know how well the resource SKUs work with extended locations.

Reply:
On the other hand, it seems we are always listing by location name it seems, so hopefully this supports custom locations as well.